tanner/utils/docker_helper.py

```diff
@@ -59,8 +59,8 @@ def __init__(self):
         execute_result = None
         try:
             container.start()
-            execute_result = container.exec_run(['sh', '-c', cmd]).decode('utf-8')
+            execute_result = container.exec_run(['sh', '-c', cmd])
             container.kill()
         except docker.errors.APIError as server_error:
             self.logger.exception('Error while executing command %s in container %s', cmd, server_error)
-        return execute_result
+        return execute_result.output.decode('utf-8')
```

Comment:
This is giving me error.

Reply:
Why did you change it in the first place?

Reply:
It is according to latest docker upgrade.